techniques/applications/systemUpdateCampaign/1.0/modules/test/test_system_update.py

```diff
@@ -0,0 +1,177 @@
+import unittest
+import json
+import tempfile
+from system_update import *
+import shutil
+import time
+
+
+class TestGeneric(unittest.TestCase):
+    def test_store_get(self):
+        tmpdir = tempfile.mkdtemp()
+        yum = Yum('id', workdir=tmpdir)
+        my_list = ['a', 'b', 'c']
+        yum.store_file('test', my_list)
+        with open(os.path.join(tmpdir, 'id_test.json'), 'r') as file:
+            content = file.read()
+            self.assertEqual(content, '["a", "b", "c"]')
+        get = yum.get_file('test')
+        self.assertEqual(get, my_list)
+        shutil.rmtree(tmpdir)
+
+    def test_lock(self):
```

Comment:
I am not sure of why the date is compared here. It does not seem to be the case in the module which only check if the file is a well formed json.